crates/orchestrator/src/network.rs

```diff
@@ -312,7 +312,11 @@ impl<T: FileSystem> Network<T> {
 
     /// Add a new parachain to the running network
     ///
-    /// NOTE: para_id must be unique in the whole network.
+    /// # Arguments
+    /// * `para_config` - Parachain configuration to deploy
+    /// * `custom_relaychain_spec` - Optional path to a custom relaychain spec to use
+    /// * `custom_parchain_fs_prefix` - Optional prefix to use when artifacts are created
```

Comment:
What does it means, isn't this leaking the way we add a parachain ? I mean, the outside world does needs to know about artifacts ?